docs/en/couchbase_client.md

```diff
@@ -42,53 +165,67 @@ if (!couchbase_client.InitCouchbase(connection_string, username, password)) {
 Use `Add` to insert a new document. This operation will fail if the document already exists:
 
 ```cpp
-std::string user_data = R"({"name": "John Doe", "age": 30, "email": "john@example.com"})";
-
-if (couchbase_client.CouchbaseAdd("user::john_doe", user_data, "my_bucket")) {
-    LOG(INFO) << "Document added successfully";
-} else {
-    LOG(ERROR) << "Failed to add document (may already exist)";
-}
+  std::string user_data = R"({"name": "John Doe", "age": 30, "email": "john@example.com"})";
+  auto add_response = couchbase_client.CouchbaseAdd("user::john_doe", user_data, FLAGS_bucket);
+  if (add_response.success) {
+    std::cout << "User data added successfully";
+  } else {
+    //based on the returned error do actual error handling
+    if(add_response.err.ec() == couchbase::errc::key_value::document_exists){
+        //handle the error
```

Comment:
Much better-- it's great when these kinds of examples show at least rudimentary error handling.